contrib/qpl-cmake/doc/build.md

```diff
@@ -0,0 +1,14 @@
+This doc gives specfic build tips for QPL_Deflate codec based on Clickhouse generic [build instructions](https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/development/build.md)
```

Comment:
If this file is moved into the regular ClickHouse documentation (please see my below comment), we should replace the link from an absolute to a relative link. As an example for the necessary syntax, please see file "docs/en/sql-reference/functions/type-conversion-functions.md".